python/interpret_community/mimic/mimic_explainer.py

```diff
@@ -677,3 +702,25 @@ def __setstate__(self, state):
         """
         self.__dict__.update(state)
         self._logger = logging.getLogger(__name__)
+
+    def _get_surrogate_model_replication_measure(self, training_data):
```

Comment:
it would be nice to figure out how to make these methods public to general users as well, since I would imagine normal SDK users might want to have access to these awesome functions too

Reply:
and maybe even add a notebook showcasing this nice functionality?

Reply:
Thanks for the idea on putting out a notebook. Let me work on a notebook and then we can decide which functions we could make public. I don't want to make a lot of functions public now, given that I have a few other PRs https://github.com/interpretml/interpret-community/pull/355 pending. For now, we can have these functions as internal.